config.json

```diff
@@ -677,6 +677,14 @@
         "prerequisites": [],
         "difficulty": 5
       },
+      {
+        "slug": "two-bucket",
+        "name": "Two Bucket",
+        "uuid": "20c471f8-e45e-4313-a23a-85d4a36c7185",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
```

Comment:
I think this exercise is quite hard and since it uses structs and enums I think perhaps the difficulty should be put as 6.

Reply:
Good thought, fixed!